webapp/channels/src/utils/constants.tsx

```diff
@@ -1391,6 +1391,18 @@ export const DefaultRolePermissions = {
     ],
 };
 
+// ModeratedPermissions are permissions that can be turned off for members and guests
+// on a per channel basis. These permissions are on by default for team/channel admins.
+export const ModeratedPermissions = [
```

Comment:
Do we not need to add here the bookmarks permissions that can also be moderated on a per channel basis?
![Screenshot 2024-09-09 at 14 59 45](https://github.com/user-attachments/assets/abdd4b08-b057-4e83-b50b-9b9fae6bb151)

Reply:
Yep...missed those.